x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/reducer.ts

```diff
@@ -199,6 +203,29 @@ export const validateAdvancedEditor = (state: State): State => {
         message: '',
       });
     }
+
+    const numTopFeatureImportanceValues = getNumTopFeatureImportanceValues(jobConfig.analysis);
+    if (
+      numTopFeatureImportanceValues !== undefined &&
```

Comment:
Ideally we should also check that they only input integers, and not e.g. `4,567` :)

![image](https://user-images.githubusercontent.com/7405507/78356004-aadf4000-75a6-11ea-9f1a-39b68f0046f0.png)

Reply:
Pushed another fix including unit tests in 6ad3355.